build/package/Microsoft.DotNet.Cli.Archive.targets

```diff
@@ -20,19 +21,33 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="GenerateArchives" 
-          DependsOnTargets="Init;Layout;SetupGenerateArchivesInputsOutputs" 
-          Inputs="%(GenerateArchivesInputsOutputs.Inputs)" 
+  <Target Name="GenerateArchives"
+          DependsOnTargets="Init;Layout;SetupGenerateArchivesInputsOutputs"
+          Inputs="%(GenerateArchivesInputsOutputs.Inputs)"
           Outputs="%(GenerateArchivesInputsOutputs.Outputs)">
 
-    <ArchiveDirectory
-        FileName="%(GenerateArchivesInputsOutputs.OutFileName)"
-        OutputDirectory="$(ArchiveOutputDirectory)"
-        InputDirectory="%(GenerateArchivesInputsOutputs.InputDirectory)" >
+    <PropertyGroup>
+      <GenerateArchivesDestinationArchive>
+        $(ArchiveOutputDirectory)/%(GenerateArchivesInputsOutputs.OutFileName).tar.gz
```

Comment:
my last comment doesn't show up for some reason

but using $(ArchiveExtension) you can avoid needing to hard code the extension values here.